molgenis-api-tests/src/test/resources/org/molgenis/api/tests/data/v3/deleteResourceCollection.json

```diff
@@ -1,6 +1,6 @@
 {
   "links": {
-    "self": "${baseUri}/api/data/v3_MyDataset"
+    "self": "${baseUri}/api/data/v3_MyDataset?size=100&page=0"
```

Comment:
How about either:
- using exactly the URL that the user entered
- not including query parameters when the value is the default value